### PR TITLE
Better exit codes

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Split exit codes and mention them in the README

Signed-off-by: lachmanfrantisek <flachman@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,14 @@ WARN:Label 'io.openshift.expose-services' has to be specified.
...
 ```
 
+### Exit codes
+
+Colin can exit with several codes:
+
+- `0` --> OK
+- `1` --> error in the execution
+- `2` --> CLI error, wrong parameters
+- `3` --> at least one failed check
```
Comment (TomasTomecek) [inline README.md]: grammar nit: *at least one check failed*

Comment (lachmanfrantisek) [reply README.md]: Corrected, thanks.

File: colin/cli/colin.py [hunk 2]
```diff
@@ -85,7 +85,7 @@ def check(target, ruleset, ruleset_file, debug, json, stat, tag, verbose):
         if not results.ok:
             sys.exit(1)
         elif results.fail:
-            sys.exit(2)
+            sys.exit(3)
```
Comment (phracek) [inline colin/cli/colin.py]: @lachmanfrantisek What about to add there a comment why exit(2) is not used? I would prefer to comment all exit codes. Does it make sense to you?

Comment (lachmanfrantisek) [reply colin/cli/colin.py]: Isn't the README enough?

Comment (phracek) [reply colin/cli/colin.py]: Yeah, let's leave it, just digging.

**Comment by phracek** [comment]
LGTM. from my point of view.

**Pull request merged**